src/AppBundle/Entity/Report/Traits/ReportProfDeputyCostsEstimateTrait.php

```diff
@@ -32,6 +33,20 @@
      */
     private $profDeputyEstimateCostTypeIds;
 
+    /**
+     * @var float
+     *
+     * @JMS\Type("double")
+     * @JMS\Groups({"prof-deputy-management-costs"})
+     */
+    private $profDeputyManagementCosts;
+
+    /**
+     * @JMS\Type("array")
+     * @JMS\Groups({"prof-deputy-management-costs"})
+     */
+    private $profDeputyManagementCostTypeIds;
+
```

Comment:
not needed

Reply:
this IS needed. Not sure the comment above is relevant now

Reply:
OK I've reinstated along with getters/setters - but just checking what this will be used for...it looked like this was linked with multi-field forms?

Reply:
Its the client side property that stores the api side Virtual property (default list of all cost Ids)